src/config/mira-config.ts

```diff
@@ -194,14 +196,19 @@ class MiraConfigClass {
 
   public getCICDConfig (): CicdConfig {
     const output = this.getFullCiProps(CONFIG_KEYS.CICD)
-    const {
-      name,
-      owner
-    } = parseGitUrl(output.repositoryUrl)
+    const githubValues: { repositoryName?: string, repositoryOwner?: string } = {}
+    if (output.provider !== 'codecommit') {
```

Comment:
This might be painful to test considering, but just wondering if this was tested w GitHub vs. CodeCommit?

Reply:
No, the test with github wasn't done. But I think it worth to do be done.